src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random RANDOM = new Random();
+    private static final ColorSupplier SUPPLIER = new ColorSupplier();
```

Comment:
Don't use static methods or static fields in your solution unless it's a constant. Static fields can lead to issues when multiple instances of the class are used concurrently. Instead, you can create an instance of Random and ColorSupplier in the constructor of the Lottery class.